src/ui/dashboardpage.h

```diff
@@ -30,9 +30,16 @@ class DashboardPage {
         Layout
     };
     void draw(App *app, Dashboard *Dashboard, Mode mode = Mode::View) noexcept;
+    void drawPlots(App *app, DigitizerUi::DashboardPage::Mode mode, Dashboard *dashboard);
+    void drawGrid(float w, float h);
     void drawLegend(App *app, Dashboard *dashboard, const Mode &mode) noexcept;
+
+private:
+    static bool plotButton(App *app, std::string_view glyph, std::string_view tooltip = {});
```

Comment:
`plotButton(...)` is short enough and could be inlined (N.B. we aim for a header-only style wherever possible).

Since there is not much state required in `plotButton(..)` and it's already marked static, maybe make this a free-standing function?